Doc/c-api/gcsupport.rst

```diff
@@ -228,3 +228,36 @@ garbage collection runs.
    Returns the current state, 0 for disabled and 1 for enabled.
 
    .. versionadded:: 3.10
+
+
+Querying Garbage Collector State
+--------------------------------
+
+The C-API provides the following inerface for querying information about
+the garbage collector.
+
+.. c:function:: void PyUnstable_GC_VisitObjects(gcvisitobjects_t callback, void *arg)
+
+   Run supplied callback on all live GC-capable objects.
```

Comment:
Format parameters correctly by wrapping them with a pair of \*.
Also note that *arg* is passed to the callback.
(I made a clumsy suggestion to improve this; feel free to improve my suggestion. Perhaps a new sentence is better.)

```suggestion
   Run *callback*, with *arg* passed, on all live GC-capable objects.
```